spec/request/api/v1/topic/topic_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Topic /api/v1/topics', type: :request do
+  let(:user) { create(:user, confirmed_at: Time.zone.today) }
+  subject do
+    10.times { create(:topic) }
+    login
+  end
+
+  it 'should return a successful response' do
+    subject
```

Comment:
I think the subject is in fact the endpoint, not the creation of the topics. The list of topics should be created with `let!`.

See https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let